packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -285,13 +371,53 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
         className: chatMessageSlotClassNames.actionMenu,
         styles: resolvedStyles.actionMenu,
       }),
+      overrideProps: {
+        id: actionMenuId.current,
+      },
     });
 
-    if (!actionMenuElement) {
-      return actionMenuElement;
+    const content = actionMenuElement ? <Ref innerRef={actionsMenuRef}>{actionMenuElement}</Ref> : actionMenuElement;
+
+    return inlineActionMenu || !content ? content : <PortalInner>{content}</PortalInner>;
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    // taken from https://github.com/microsoft/fluentui/pull/17329
+    const focusableElements = actionsMenuRef?.current?.querySelectorAll(
```

Comment:
nit: I don't think the ref itself can disappear 

```suggestion
    const focusableElements = actionsMenuRef.current?.querySelectorAll(
```